codemp/server/sv_main.cpp

```diff
@@ -598,8 +598,7 @@ void SVC_Info( netadr_t from ) {
 	Info_SetValueForKey( infostring, "mapname", sv_mapname->string );
 	Info_SetValueForKey( infostring, "clients", va("%i", count) );
 	Info_SetValueForKey( infostring, "g_humanplayers", va("%i", humans) );
-	Info_SetValueForKey( infostring, "sv_maxclients",
-		va("%i", sv_maxclients->integer - sv_privateClients->integer ) );
+	Info_SetValueForKey( infostring, "sv_maxclients", va("%i", sv_maxclients->integer ) );
```

Comment:
We should keep max clients the same, but just bump up the number of clients to include the private users. That way you can see how many people are on the server. Thus a full 14 player server with 2 private clients will show as 16/14.